Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Dashboard/Management/Driver.hs

```diff
@@ -1217,21 +1218,21 @@ getDriverStats merchantShortId opCity mbEntityId mbFromDate mbToDate requestorId
                   }
         _ -> throwError (InvalidRequest "Invalid date range: Dates must be empty, same day, or max 7 days apart.")
 
-    isAssociationBetweenTwoPerson :: (EsqDBFlow m r, MonadFlow m, CacheFlow m r) => DP.Person -> DP.Person -> m Bool
-    isAssociationBetweenTwoPerson requestedPersonDetails personDetails = do
-      case (requestedPersonDetails.role, personDetails.role) of
-        (DP.OPERATOR, DP.DRIVER) -> checkDriverOperatorAssociation personDetails.id requestedPersonDetails.id
-        (DP.OPERATOR, DP.FLEET_OWNER) -> checkFleetOperatorAssociation personDetails.id requestedPersonDetails.id
-        (DP.FLEET_OWNER, DP.DRIVER) -> checkFleetDriverAssociation requestedPersonDetails.id personDetails.id
-        _ -> return False
```

Comment:
Are you sure that dashboard admin is the same as admin on bpp side? If not, then you can add bypass here also

`(DP.ADMIN, _) -> True`